component_event/tests/test_event.py

```diff
@@ -17,7 +17,7 @@
 
 
 @tagged("standard", "at_install")
-class TestEventWorkContext(unittest.TestCase):
+class TestEventWorkContext(TreeCase):
```

Comment:
I think it would be better to inherit from `TransactionCase` or `SavepointCase` here, if possible, to standardize the behavior... 

It would also eliminate the need of the explicit tag before, I think...

Reply:
this is wrong as well. This is a unit test, not an odoo related test.